src/assets/styles/app.css

```diff
@@ -0,0 +1,126 @@
+:root{
+    --secondary-bg: #000;
+    --secondary-fg: #fff;
+}
+.box {
+    color: white;
+    margin: 20px auto;
+    /*padding: 10px;*/
+    text-align: center;
+    line-height: 1.5;
+}
+
+.fixed-box {
+    width: 200px;
+    height: 100px;
+    background-color: #3366ff;
+}
+
+header, footer {
+    background-color: var(--secondary-bg);
+    color:var(--secondary-fg) ;
+    
+} 
+nav a, footer a{
+    color:var(--secondary-fg);
+    background-color: rgb(0, 0, 0);
+    text-decoration: none;
+}
+nav li {
+    list-style-type: none;
+    display: inline-block;
+}
+nav ul {
+    padding: 0;
+    font-size:20px;
+    font-family: Impact, Haettenschweiler, sans-serif;
```

Comment:
The `Haettenschweiler` font is not included in the site. It would thus only work for people who have it installed on their device (same for Impact, but most people will have that one).